dataCollection/example/example.py

```diff
@@ -0,0 +1,61 @@
+
+from dataCollection.airports.SanDiego import SanDiego
+from dataCollection.airports.Chicago import Chicago
+from dataCollection.actualData.countExits import countTotals 
+from dataCollection.plotting.plotUtils import openPickle
+from dataCollection.plotting.plotting import (
+    plotMap,
+    plotMultipleTrails,
+    plotExitBoxes,
+)
+import os
+
+SAN = SanDiego()
+ORD = Chicago()
+
+allSAN = os.path.join(os.path.dirname(__file__), "sampleData/SAN")
```

Comment:
Duplicating `os.path.dirname(__file__)` is an unacceptable coding practice for this carefully maintained and documented repository that is widely used in academia

Reply:
ok bud